COLLADA2GLTF/COLLADA2GLTFWriter.cpp

```diff
@@ -1463,7 +1463,7 @@ namespace GLTF
 
         //
         shared_ptr <GLTFBufferView> weightsView = createBufferViewWithAllocatedBuffer(weightsPtr, 0, skinAttributeSize, true);
-        shared_ptr <GLTFAccessor> weightsAttribute(new GLTFAccessor(profile, "FLOAT", GLTFUtils::getTypeForVectorSize(bucketSize)));
+        shared_ptr <GLTFAccessor> weightsAttribute(new GLTFAccessor(profile, "FLOAT", GLTFUtils::getTypeForVectorSize((int)bucketSize)));
```

Comment:
It's probably better to just declare the variable as an int instead of a size_t, as opposed to casting it everywhere.